packages/terra-date-time-picker/src/terra-dev-site/test/date-time-picker/twelve-hour/DateTimePickerValue12HourMobile.test.jsx

```diff
@@ -52,22 +46,6 @@ export default class DateTimePicker12HourMobile extends React.Component {
           value={this.state.datetime}
           timeVariant={DateTimeUtil.FORMAT_12_HOUR}
         />
-
-        <h3 id="titleWithDateTimeValue">Disabled Date-Time-Picker</h3>
-        <DateTimePickerExampleTemplate
-          id="disabledDatetimeValueProvided"
-          disabled
-          value={this.state.datetime}
-          timeVariant={DateTimeUtil.FORMAT_12_HOUR}
-        />
-
-        <h3 id="titleWithDateTimeValue">Date-Time-Picker with Seconds</h3>
```

Comment:
The test example with seconds seems to have been deleted and not just moved elsewhere. It didn't have any dedicated screenshots in the spec file but was captured in other screenshots.

Reply:
Added the test example in separate file in  009e0a3